### ISSUE TITLE
matt2020: script to update the version number

### ISSUE THREAD

**Issue by MatthiasWM** [pguyot/Einstein#78]
Keeping the version number up to date on all platforms drives me nuts. Write something!

**Comment by MatthiasWM** [comment]
Version number is now in CMakeLists.txt and is copied into the internal Rex and the About window.

We need to add the version information to the various platform resources as well.

**Comment by splorp** [comment]
Related to this, NewtTest shows macOS Einstein 2020.3.5 as simply 2020.3 … should the “build” number be included in the versioning being reported?

**Comment by MatthiasWM** [comment]
Yes, thanks. I generate that number by patching the Rex, and it only has room for 7 characters, or I have to recompile the Rex. I'll do that eventually.

**Comment by splorp** [comment]
Great! Thanks, Matthias.

**Comment by MatthiasWM** [comment]
Add a NewtonScript function to return more information about Einstein and its host. Update the Version number returned by NewtTest @FrankFromPdaSoft
